app/styles/app.scss

```diff
@@ -291,4 +301,41 @@ form {
   }
 }
 
+.show-poster {
+  min-height: 500px;
+}
+
+.show-info {
+  padding: 1em;
+  border-image: repeating-linear-gradient(
+    90deg,
+    $green,
+    $green 20px,
+    $pink 20px,
+    $pink 40px,
+    $yellow 40px,
+    $yellow 60px,
+    $blue 60px,
+    $blue 80px
+  );
```

Comment:
Nice :rainbow:

Reply:
This part might be hard to convert to tailwind syntax, but all the other stuff may be possible e.g. what I did with the `cool-button` class:
https://github.com/datafruits/datafruits/blob/master/app/styles/app.scss#L104

http://tailwindcss.com/docs/